src/stories/PageLayout.stories.tsx

```diff
@@ -15,36 +17,36 @@ const Template: StoryFn<typeof PageLayout> = args => (
   </PageLayout>
 )
 
-const emeraldRoute = {
-  routePath: '/:layer',
-  routeParams: { layer: 'emerald' },
+const sapphireRoute = {
+  routePath: '/:network/:layer',
+  routeParams: { network: Network.mainnet, layer: Layer.sapphire },
 }
```

Comment:
Funny thing, I think this is char by char the same as my (unfinished) https://github.com/oasisprotocol/explorer/pull/502/files.